commit 8bafdc384d7f02fe172a24602d3083c84481a143
Author: Laurent Montel <montel@kde.org>
Date:   Tue Mar 14 12:45:14 2023 +0100

    Use correct INSTALL_TARGETS_DEFAULT_ARGS
    
    Non-KF targets must use KDE_INSTALL_TARGETS_DEFAULT_ARGS. The KF_
    variables adds /include/KF[56] to
    INTERFACE_INCLUDE_DIRECTORIES and that causes cmake errors for users
    of this library if /include/KF[56] doesn't exist
    (e.g. because one uses KF libraries provided by the distro).

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index a3a209f..7f902e1 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -176,7 +176,7 @@ set_target_properties(KPim${KF_MAJOR_VERSION}TextEdit PROPERTIES
     EXPORT_NAME PimTextEdit
     )
 
-install(TARGETS KPim${KF_MAJOR_VERSION}TextEdit EXPORT KPim${KF_MAJOR_VERSION}TextEditTargets ${KF_INSTALL_TARGETS_DEFAULT_ARGS})
+install(TARGETS KPim${KF_MAJOR_VERSION}TextEdit EXPORT KPim${KF_MAJOR_VERSION}TextEditTargets ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 ########### Generate Headers ###############
 ecm_generate_headers(KPimTextEdit_CamelCase_HEADERS
